system-int-test/test/cmr/system_int_test/ingest/service_ingest_test.clj

```diff
@@ -0,0 +1,99 @@
+        ; (service-util/assert-service-saved service-data
+        ;                                    "umm-var-user1"
+        ;                                    concept-id
+        ;                                    revision-id)
+        ))))
+
+(deftest create-service-ingest-permissions-test
+  (testing "ingest create service permissions"
+    (let [;; Groups
+          guest-group-id (e/get-or-create-group
+                          (s/context) "umm-var-guid1")
+          reg-user-group-id (e/get-or-create-group
+                             (s/context) "umm-var-guid2")
+          update-group-id (e/get-or-create-group
+                           (s/context) "umm-var-guid3")
+          ;; Tokens
+          guest-token (e/login
+                       (s/context) "umm-var-user1" [guest-group-id])
+          reg-user-token (e/login
+                          (s/context) "umm-var-user1" [reg-user-group-id])
+          update-token (e/login
+                        (s/context) "umm-var-user2" [update-group-id])
+          ;; Grants
+          guest-grant-id (e/grant
+                          (assoc (s/context) :token guest-token)
+                          [{:permissions [:read]
+                            :user_type :guest}]
+                          :system_identity
+                          {:target nil})
+          reg-user-grant-id (e/grant
+                             (assoc (s/context) :token reg-user-token)
+                             [{:permissions [:read]
+                               :user_type :registered}]
+                             :system_identity
+                             {:target nil})
+          update-grant-id (e/grant-group-admin
+                           (assoc (s/context) :token update-token)
+                           update-group-id
+                           :update)
+          service-data (service-util/make-service)]
+      (testing "acl setup and grants for different users"
+        (is (e/not-permitted? guest-token
+                              guest-grant-id
+                              guest-group-id
+                              (ingest-util/get-ingest-update-acls guest-token)))
+        (is (e/not-permitted? reg-user-token
+                              reg-user-grant-id
+                              reg-user-group-id
+                              (ingest-util/get-ingest-update-acls reg-user-token)))
+        (is (e/permitted? update-token
+                          update-grant-id
+                          update-group-id
+                          (ingest-util/get-ingest-update-acls update-token))))
+      (testing "ingest service creation permissions"
+        (are3 [token expected]
+          (let [response (service-util/create-service token service-data)]
+            (is (= expected (:status response))))
+          "System update permission allowed"
+          update-token 201
+          "Regular user denied"
+          reg-user-token 401
+          "Guest user denied"
```

Comment:
Looks like this test is also missing from `variable_ingest_test.clj` -- will add there too.